packages/dds/tree/src/shared-tree/treeApiAlpha.ts

```diff
@@ -203,7 +203,12 @@ export const TreeAlpha: {
 		compressedData: JsonCompatible<IFluidHandle>,
 		options: { idCompressor?: IIdCompressor } & ICodecOptions,
 	): Unhydrated<TreeFieldFromImplicitField<TSchema>>;
-} = {
+}
+
+/**
+ * @sealed @alpha
```

Comment:
Nit: if we aren't going to document this independently anyways, shall we add an `@inheritDoc` pointing to the interface?